v2.0/app/containers/MassEnergizeSuperAdmin/Community/Profile/About.js

```diff
@@ -112,7 +112,19 @@ class About extends React.Component {
     if (community !== 0) {
       const goal = community.goal ? community.goal : {};
       const targ = goal.target_number_of_households;
-      const att = goal.attained_number_of_households;
+      const att = goal.attained_number_of_households + goal.organic_attained_number_of_households;
```

Comment:
if goal was {} then you will be adding two `null` 's which would not be good lol.  Maybe do something like

```
(goal.attained_number_of_households || 0) + (goal.organic_attained_number_of_households || 0)
```